l10n_ar_purchase/i18n/es.po

```diff
@@ -83,3 +83,14 @@ msgstr "Pedido de compra"
 #: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.report_purchasequotation_document
 msgid "Request for Quotation"
 msgstr "Solicitud de presupuesto"
+
+#. module: l10n_ar_purchase
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchasequotation
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchaseorder
+msgid "Date due:"
+msgstr "Fecha de vencimiento"
+
+#. module: l10n_ar_purchase
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchaseorder
+msgid "Approve date:"
+msgstr "Fecha de aprobacion"
```

Comment:
re order this terms where they should be.
To do this you can load this into a local database for example and then using Odoo tools export again the translations. apply for both files